ansible/roles/docker_ce_install/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
You appear to be missing anything that invokes Ubuntu.20.yml

Reply:
I only support UB20 for the role at this time..
If anyone calls this for any other linux it will puke.. Is that what you mean?
I did add a README.

Reply:
I 2x checked my cloned tree.. Looks like all the files are checked in.
I did re-run the play and I do have a running Instana on a new host...